src/components/form/form.component.tsx

```diff
@@ -14,7 +13,36 @@ import {
   StyledFullWidthButtons,
 } from "./form.style";
 
-const Form = ({
+export interface FormProps extends SpaceProps {
+  /** Alignment of buttons */
+  buttonAlignment?: "left" | "right";
```

Comment:
What do you think about importing the `FORM_BUTTON_ALIGNMENTS` from `form.config.ts` to create a type here?